CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### NEXT
+
+- Worker: RtcLogger: Use `MS_DUMP()` instead of `std::cout()` ([PR #1553](https://github.com/versatica/mediasoup/pull/1553)).
```

Comment:
Being a bit pedantic I don't get this syntax:

```
Foo: Bar: xxx
```

```suggestion
- `RtcLogger`: Use `MS_DUMP()` instead of `std::cout()` ([PR #1553](https://github.com/versatica/mediasoup/pull/1553)).
```

No need to mention Worker. Everything related to RTC is in the worker already.
Said that, IMHO we don't need an entry in CHANGELOG for this change. This is not public API and it's only intended for pure debugging.